extensions/standard-processors/processors/ListenSyslog.cpp

```diff
@@ -53,15 +53,15 @@ const core::Property ListenSyslog::ParseMessages(
 const core::Property ListenSyslog::MaxQueueSize(
     core::PropertyBuilder::createProperty("Max Size of Message Queue")
         ->withDescription("Maximum number of Syslog messages allowed to be buffered before processing them when the processor is triggered. "
-                          "If the buffer full, the message is ignored. If set to zero the buffer is unlimited.")
-        ->withDefaultValue<uint64_t>(0)->build());
+                          "If the is buffer full, the message is ignored. If set to zero the buffer is unlimited.")
```

Comment:
the "is" is probably not where intended

Reply:
Good catch, fixed in 2798590a9068fcdf1061f4284f30e3454385ec0c